Add a new action with customized vibration (#28)
GDevelopApp/GDevelop-documentation: docs/gdevelop5/all-features/gamepad.md

Each gamepad connected to the computer or phone is numbered from 1 to 4:

* If your game is single player, you'll usually always use condition for the Gamepad 1.
* If your game is multi player, it's a good idea to use the **Gamepad connected** condition to check which gamepads are connected. You could store in a variable the gamepad number associated to a player. You can also force the first player to use the Gamepad 1, and the other player to use Gamepad 2.

![](/gdevelop5/all-features/gamepad-condition-connected.png)

## Vibrate a gamepad

Use the action **Gamepad vibration** to make a gamepad vibrate.

> **Advanced Gamepad vibration** allows to setup the magnitude of the vibrations with a lower and a higher bound for motors rumble.

Reviewer: Sorry, actually, I'm not sure about the change I suggested.
What are the low and high values? Is it 2 motors: one for small high frequency vibrations and one for big low frequency ones?

Author: From the [w3c spec](https://w3c.github.io/gamepad/extensions.html#gamepadhapticeffecttype-enum):

> [strongMagnitude](https://w3c.github.io/gamepad/extensions.html#dom-gamepadeffectparameters-strongmagnitude) and [weakMagnitude](https://w3c.github.io/gamepad/extensions.html#dom-gamepadeffectparameters-weakmagnitude) set the vibration intensity levels for the low-frequency and high-frequency rumble motors, normalized to the range [0,1], default 0. An effect with magnitude values outside of this range is not a [valid effect](https://w3c.github.io/gamepad/extensions.html#dfn-valid-effect).

Author: ```suggestion
**Advanced Gamepad vibration** allows to setup the strong magnitude and weak magnitude set the vibration intensity levels for the low-frequency and high-frequency rumble motors.
``````

> **Advanced Gamepad vibration** allows to setup the magnitude of the vibrations with a lower and a higher bound for motors rumble.

Reviewer: ```suggestion
**Advanced Gamepad vibration** allows to setup vibration magnitudes for low and high frequency rumble motors.
```

> **Advanced Gamepad vibration** allow you to setup the magnitude of the vibrations with two new values, the low and high value for the motors rumble.

Reviewer: ```suggestion
**Advanced Gamepad vibration** allows to setup the magnitude of the vibrations with a lower and a higher bound for motors rumble.
```
